### DISCUSSION TITLE
开源之道（2016.5.28） - 「开源之道」

### DISCUSSION THREAD

**Discussion by giscus** [OCselected/ocselected.github.io#104] [Announcements]
# 开源之道（2016.5.28） - 「开源之道」

https://opensourceway.community/posts/opensource/daily_reading/2016/2016_5/2016_5_28/

**Comment by ZoomQuiet** [comment]
> 是也乎,(￣▽￣)

其实这种日报类吐糟,坚持下来的意义, 就是拥有了自己专用的可环比/ 对比的时间线...
很多当初的想法就可以进行自我补刀了...
